cmd/zapi/cmd/rename/rename.go

```diff
@@ -29,18 +29,34 @@ type Command struct {
 }
 
 func (c *Command) Run(args []string) error {
-	if len(args) != 2 {
-		return errors.New("expected <old_name> <new_name>")
-	}
-	oldname := args[0]
-	newname := args[1]
-	id, err := cmd.GetSpaceID(c.Context(), c.Connection(), oldname)
-	if err != nil {
+	defer c.Cleanup()
+	if err := c.Init(); err != nil {
 		return err
 	}
+
+	var err error
+	var id api.SpaceID
+	var newname string
+	switch len(args) {
```

Comment:
zapi supports global options `-s` and `-id` (ie, `zapi -s <space> <subcommand>`) that can be used to specify the target space of a command. So the 1 argument handling allows relying on those options to to specify the space to be renamed.  And it may not be clear due to the size of the PR, but I put this support in so I could write a ztest that verifies the 'UpdateSpaceName' api properly checks permissions.